### ISSUE TITLE
suggested rest endpoints for batch update and batch delete

### ISSUE THREAD

**Issue by gebrits** [balderdashy/sails#1805]
afaik batch update and batch delete aren't supported by crud blueprints so I'd like to add them to my controllers. 

As this must have been done before: what are considered best practices for modelling batch update and batch delete in a restful manner? I realize this isn't sailsJS related per se but still useful imo.

I was thinking the following for *batch delete*: 

`DELETE /collection`
payload: `['idA','idB']`

*batch update* could go multiple ways. Here's one way: 

`PUT /collection`
payload: `[['idA','idB'], {//partial update here}]`

Looking for best practices. Thanks.

**Comment by mpduffey** [comment]
+1 to the blueprints

**Comment by mikermcneil** [comment]
Thanks for posting, @gebrits.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
